woocommerce/woocommerce
# Remove unused "@ts-expect-error" directives from github integration

Removes unused directives from tools/monorepo-utils/src/core-github/repo.ts which were causing `pnpm install` to fail.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





* Running `pnpm install` was failing due to there being two unused directives in `tools/monorepo-utils/src/core-github/repo.ts`.

![CleanShot 2023-11-21 at 14 17 29@2x](https://github.com/woocommerce/woocommerce/assets/3846331/66808faa-b5ba-465c-9d6f-7976ec6b7e32)


* This PR removes them so that `pnpm install` runs as expected.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. `git clone` this branch into a fresh directory
2. Set your nvm environment using `nvm use`
3. From the base directory of the monorepo, run `pnpm install`
4. Verify that the install ran as expected and you did not encounter any errors.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Fix TS2578 errors on pnpm install

#### Comment 

</details>
